launcher-implementation/src/main/input_resources/sbt/sbt.boot.properties

```diff
@@ -12,8 +12,8 @@
 
 [repositories]
   local
+  jcenter: https://jcenter.bintray.com/
 ${{repositories}}
-  maven-central
```

Comment:
Which are those?

Also this is just a template.  It gets replaced when bundled by sbt or other projects (I need to write the readme on how after I finish merging this into sbt/sbt), so any changes to the properties here are just cosmetic.

Primary thing in th patch is the new built in enum for 1.0